splink/linker.py

```diff
@@ -2080,69 +2080,36 @@ def cluster_pairwise_predictions_at_threshold(
         return cc
 
     def profile_columns(
-        self, column_expressions: str | list[str] = None, top_n=10, bottom_n=10
+        self,
+        column_expressions: str | list[str],
+        top_n=10,
+        bottom_n=10,
+        distribution_plots=True,
     ):
-        """
-        Profiles the specified columns of the dataframe initiated with the linker.
-
-        This can be computationally expensive if the dataframe is large.
-
-        For the provided columns with column_expressions (or for all columns if
-         left empty) calculate:
-        - A distribution plot that shows the count of values at each percentile.
-        - A top n chart, that produces a chart showing the count of the top n values
-        within the column
-        - A bottom n chart, that produces a chart showing the count of the bottom
-        n values within the column
-
-        This should be used to explore the dataframe, determine if columns have
-        sufficient completeness for linking, analyse the cardinality of columns, and
-        identify the need for standardisation within a given column.
```

Comment:
Did you mean to delete the docstring?

Reply:
I did. I have added the docstring to the base function within the base function profile_columns. These functions sit as wrappers, so would involve duplicating the doc string. Happy to do so if needed for markdown docs to function correctly, but is it needed?